packages/apollo-server-core/src/runHttpQuery.ts

```diff
@@ -218,9 +250,16 @@ export async function runHttpQuery(
         return Promise.reject(e);
       }
 
-      return Promise.resolve({ errors: [formatErrorFn(e)] });
+      return Promise.resolve({
```

Comment:
The `Promise.resolve` here can be removed since an `async` function implicitly wraps return values in `Promise.resolve`.